source/guide_typo3-cms.rst

```diff
@@ -0,0 +1,165 @@
+ [isabell@stardust isabell]$ composer create-project typo3/cms-base-distribution typo3-cms
+ Installing typo3/cms-base-distribution (v9.5.0)
+   - Installing typo3/cms-base-distribution (v9.5.0)
+ Created project in typo3-cms
+ Loading composer repositories with package information
+ Updating dependencies (including require-dev)
+ Package operations: 76 installs, 0 updates, 0 removals
+   - Installing typo3/cms-composer-installers (v2.2.1)
+   - Installing typo3/class-alias-loader (1.0.1)
+ [...]
+ [isabell@stardust isabell]$
+
+Now remove the `document root`_ and create a symlink to the ``public`` directory.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ rm -rf html
+ [isabell@stardust isabell]$ ln -s /var/www/virtual/$USER/typo3-cms/public html
+ [isabell@stardust isabell]$
+
+Configuration
+=============
+
+Step 1
+------
+
+Point your browser to your website URL and append ``/typo3`` (e.g. isabell.uber.space/typo3). You will be greeted with a
+"Thank you for choosing TYPO3" message.
+
+Create an empty file ``FIRST_INSTALL`` (no file extension, all uppercase) inside your document root and reload the page
+in your browser. You will be redirected to the TYPO3 Install Tool which will guide you through the remaining steps.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html
+ [isabell@stardust html]$ touch FIRST_INSTALL
+
+.. note:: In case you have any problems in your environment, you will get warnings or hints in this screen. In this case, you should try to fix them.
+
+For the purpose of this guide we assume there are none, so we can proceed with the next step.
+
+Step 2
+------
+
+Enter your database credentials_, keep the other settings unchanged, they are correct as they are.
+
+Step 3
+------
+
+Create an additional_ database - for example: ``isabell_typo3``.
+
+Step 4
+------
+
+Enter a username and password for your first TYPO3 admin user (the password will also be configured for the Install Tool).
+
+.. note:: For security reasons it's best to **not** use the name ``admin``.
+
+Choose a "site name" which will identify this installation (in the page tree and browser title).
+
+Step 5
+------
+
+Choose whether you want to start with an empty TYPO3 installation (no pages, templates, configuration) or if you want
+to have a preconfigured basis to start from.
+
+Next Steps
+==========
+
+The basic installation procedure is now complete, TYPO3 will be up and running and the most appropriate settings will
+have been made for you. You will get redirected to the TYPO3 Backend and can log in with the admin user account you
+created earlier.
+
+If you wish to make changes to your installation at a later stage, use the `Install Tool`_.
+
+For further details, please have a look at the official `installation guide`_.
+
+Updates
+=======
+
+Subscribe to the `TYPO3 Announce`_ mailing list to get regular updates regarding new TYPO3 releases and security bulletins.
```

Comment:
Please document how to actually update a typo3 instance installed like in this guide. The built-in update seems to be disable by default. Depending on the complexity a link to the relevant typo3 documentation article might be sufficient.